File: src/abc/153/a/Main.java
import java.text.DecimalFormat;
import java.util.*;
public class Main {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        double h = sc.nextDouble();
        double a = sc.nextDouble();

        System.out.println((new DecimalFormat("###########")).format(Math.ceil(h/a)));
    }
}

File: src/abc/153/a/main.kt
package abc.`153`.a

import java.text.DecimalFormat
import java.util.*

object Main {
    @JvmStatic
    fun main(args: Array<String>) {
        val sc = Scanner(System.`in`)
        val h = sc.nextDouble()
        val a = sc.nextDouble()
        println(DecimalFormat("###########").format(Math.ceil(h / a)))
    }
}